lib/controllers/base.js

```diff
@@ -55,18 +56,21 @@ Controller.prototype.route = function() {
 
   // NOTE: is there a better place to do this mapping?
   var pathOptions = endpoint.string;
-  if (app.name === 'restify') {
-    if (self.method === 'delete')
+  if (this.isRestify) {
+    if (self.method === 'delete') {
```

Comment:
I think the main problem here is mapping web server with `del` or `delete` function. So, we don't need the variable `this.isRestify`, we just check and map, we don't add new params at initialize, how do you think about this?